src/player.js

```diff
@@ -794,6 +794,7 @@ export default class Player extends FakeEventTarget {
         sheet.insertRule(`#${this._playerId} .${SUBTITLES_CLASS_NAME} > div > div > div { ${style.toCSS()} }`, 0);
       }
       this._textStyle = style;
+      this.dispatchEvent(new FakeEvent(CustomEvents.TEXT_STYLE_CHANGED, {textStyle: style}));
```

Comment:
Don’t emit object by ref, in this case don’t emit the event with payload at all, make the listener of this explicitly need to call player.textStyle to get it and remove this payload as this is not data but an actual class instance object

Reply:
done